tools/nebula-spark/dependency-reduced-pom.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <parent>
+    <artifactId>tools</artifactId>
+    <groupId>com.vesoft</groupId>
+    <version>1.0.1</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>nebula-spark</artifactId>
+  <build>
+    <plugins>
+      <plugin>

```

Comment:
AFAIK `dependency-reduced-pom.xml` is create by maven?

Reply:
yeah，I should remove this file.